database/mssql/scripts/dbscripts/PSP_PIMS_S24_00/Alter Up/01_PSP_PIMS_S23_01_Alter_Up.sql

```diff
@@ -0,0 +1,2510 @@
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [dbo].[BCA_FOLIO_GENERAL_VALUE] ALTER COLUMN [PROPERTY_SUBCLASS_DESCRIPTION] nvarchar(255) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.BCA_AREA_BCTRANSIT_VALUE
+PRINT N'Alter table dbo.BCA_AREA_BCTRANSIT_VALUE'
+GO
+ALTER TABLE [dbo].[BCA_AREA_BCTRANSIT_VALUE] ALTER COLUMN [PROPERTY_CLASS_DESCRIPTION] nvarchar(255) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [dbo].[BCA_AREA_BCTRANSIT_VALUE] ALTER COLUMN [PROPERTY_SUBCLASS_DESCRIPTION] nvarchar(255) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.BCA_FOLIO_SCHOOL_VALUE
+PRINT N'Alter table dbo.BCA_FOLIO_SCHOOL_VALUE'
+GO
+ALTER TABLE [dbo].[BCA_FOLIO_SCHOOL_VALUE] ALTER COLUMN [PROPERTY_CLASS_DESCRIPTION] nvarchar(255) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [dbo].[BCA_FOLIO_SCHOOL_VALUE] ALTER COLUMN [PROPERTY_SUBCLASS_DESCRIPTION] nvarchar(255) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.BCA_DATA_ADVICE_SCHOOL_VALUE
+PRINT N'Alter table dbo.BCA_DATA_ADVICE_SCHOOL_VALUE'
+GO
+ALTER TABLE [dbo].[BCA_DATA_ADVICE_SCHOOL_VALUE] ALTER COLUMN [PROPERTY_CLASS_DESCRIPTION] nvarchar(255) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [dbo].[BCA_DATA_ADVICE_SCHOOL_VALUE] ALTER COLUMN [PROPERTY_SUBCLASS_DESCRIPTION] nvarchar(255) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.BCA_AREA_SCHOOL_VALUE
+PRINT N'Alter table dbo.BCA_AREA_SCHOOL_VALUE'
+GO
+ALTER TABLE [dbo].[BCA_AREA_SCHOOL_VALUE] ALTER COLUMN [PROPERTY_CLASS_DESCRIPTION] nvarchar(255) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [dbo].[BCA_AREA_SCHOOL_VALUE] ALTER COLUMN [PROPERTY_SUBCLASS_DESCRIPTION] nvarchar(255) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.BCA_JURISDICTION_BCTRANSIT_VALUE
+PRINT N'Alter table dbo.BCA_JURISDICTION_BCTRANSIT_VALUE'
+GO
+ALTER TABLE [dbo].[BCA_JURISDICTION_BCTRANSIT_VALUE] ALTER COLUMN [PROPERTY_CLASS_DESCRIPTION] nvarchar(255) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [dbo].[BCA_JURISDICTION_BCTRANSIT_VALUE] ALTER COLUMN [PROPERTY_SUBCLASS_DESCRIPTION] nvarchar(255) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Create view dbo.PIMS_PROPERTY_BOUNDARY_VW
+PRINT N'Create view dbo.PIMS_PROPERTY_BOUNDARY_VW'
+GO
+/* ---------------------------------------------------------------------- */
+/* Add views                                                              */
+/* ---------------------------------------------------------------------- */
+
+CREATE VIEW [dbo].[PIMS_PROPERTY_BOUNDARY_VW] AS
+SELECT PROP.PROPERTY_ID
+     , PROP.PID
+     , RIGHT('000000000' + CAST(PROP.PID AS VARCHAR(9)), 9) AS PID_PADDED
+     , PROP.PIN
+     , PROP.PROPERTY_TYPE_CODE
+     , PROP.PROPERTY_STATUS_TYPE_CODE
+     , PROP.PROPERTY_DATA_SOURCE_TYPE_CODE
+     , PROP.PROPERTY_DATA_SOURCE_EFFECTIVE_DATE
+     , PROP.PROPERTY_CLASSIFICATION_TYPE_CODE  
+     , (SELECT STRING_AGG(TENURE_DESC, ', ')
```

Comment:
Hmm, what is this? is this related to Alejandro's changes?

Reply:
Yes.  It creates a comma-separated string of TENURE_DESC if more than one are returned in the subquery.  If only one is returned, the string does not get comma-terminated.